spot-price-monitor.py

```diff
@@ -80,6 +115,10 @@ def get_args():
     parser.add_argument('-r', '--region', type=str, default='us-east-1',
                         help='''The region that the cluster is running
                         in (Default: us-east-1)''')
+    parser.add_argument('--ondemand', action="store_true", default=False,
```

Comment:
Could you please change the flag name to be consistent with other multi-word flags?
```suggestion
    parser.add_argument('--on-demand', action="store_true", default=False,
```